transport-classes-io_uring/src/main/java/io/netty/channel/uring/IoUringRecvByteAllocatorHandle.java

```diff
@@ -35,13 +35,15 @@ final class IoUringRecvByteAllocatorHandle extends RecvByteBufAllocator.Delegati
         super(handle);
     }
 
+    private boolean firstRead;
```

Comment:
This field is not set to `true` anywhere.

Reply:
doh! Forgot to push... one sec